templates/oam/workloads/multiclusterlinkerdaddon_definition.json

```diff
@@ -0,0 +1,12 @@
+{
+    "apiVersion": "core.oam.dev/v1alpha1",
+    "kind": "WorkloadDefinition",
+    "metadata": {
+        "name": "MultiClusterLinkerdAddon"
+    },
+    "spec": {
+        "definitionRef": {
+            "name": "MultiClusterLinkerdAddon.meshery.layer5.io"
+        }
```

Comment:
I the spec was updated to mandate the `metadata` field. Refer: https://docs.google.com/document/d/1B2N78EdRiZF-yVo1-HY3syppwBBDumgMuYg6seD-AJ4/edit#heading=h.nwmv63agfltx